pages/instructorAssessmentInstance/instructorAssessmentInstance.js

```diff
@@ -156,6 +159,18 @@ router.post('/', (req, res, next) => {
                 res.redirect(req.originalUrl);
             });
         });
+    } else if (req.body.__action == 'regrade_question') {
+        if (!config.regradeActive) {
+            res.status(204).send();
+            return;
```

Comment:
Any specific reason to send a `204` status instead of the `return next(error.make())` call that's been used everywhere else?

Reply:
The reason I didn’t use `error.make()` is because I just want to button to basically do a noop when regrade is not active, so I think return a 204 (which is not an error) make sense(?)